scripts/windows/setup_signing_key

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+echo "$SFDX_WINDOWS_KEY" | base64 --decode > /tmp/sfdx-windows-key
```

Comment:
Is this going to run on a windows machine? Will it have access to bash?

Reply:
From slack:
![image](https://user-images.githubusercontent.com/1084688/108768755-0d765d00-7526-11eb-9261-48794c819058.png)